engine/src/main/java/org/terasology/engine/subsystem/lwjgl/Lwjgl2Sync.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2020 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.engine.subsystem.lwjgl;
+
+import org.lwjgl.glfw.GLFW;
+
+public final class Lwjgl2Sync {
```

Comment:
A brief bit of class-level javadoc is always nice to have, even if the methods are javadocced :-)